src/modules/addRemoveEdit.js

```diff
@@ -0,0 +1,78 @@
+import trash from "../assets/icons8-waste-24.png";
+
+const ul = document.querySelector(".list-items");
+let todos = JSON.parse(localStorage.getItem("todos")) || [];
+export const getToDos = () => todos;
+export const setToDos = (newTodos) => {
+  todos = newTodos;
+};
+
+export const addToDo = (todo) => {
+  const x = {
+    description: todo.description,
+    completed: todo.completed,
+    index: todo.index,
+  };
+  todos.push(x);
+  localStorage.setItem("todos", JSON.stringify(todos));
+  return x;
```

Comment:
Please assign your variables meaningful names. Meaningful names make it easier for people who read through your code to understand it.